resources/views/user/show-user-profile.blade.php

```diff
@@ -0,0 +1,178 @@
+<!doctype html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="{{ asset('css/fontaswesme-all.min.css') }}">
+    <link rel="stylesheet" href="{{ asset("css/app.css") }}">
+    <link rel="stylesheet" href="{{ asset("css/user/main.css") }}">
+</head>
+<body>
+<div class="container">
+    <div class="row mt-5 ">
+        <div class="col-lg-8 col-sm-12 mt-3   mx-auto ">
+            <div class="row bg-light shadow-sm first-card p-3 card-profile-margin">
+
+                    <div class="col-lg-2 col-sm-12 my-auto d-flex justify-content-center ">
+                        <img src="{{asset('avatar/'.$user->image)}}" width="110px" height="110px"
+                             class="rounded-circle">
+                    </div>
+                    <div class="col-lg-10 col-sm-12  pt-2  ">
+                        <h3 class="font-weight-bold hover-edit">{{$user->first_name}}  {{$user->last_name}} <a href="#" class="edit-ancor">
+                                <i class="fas fa-pencil-alt edit-fontawsom"></i></a></h3>
+                        <h6>{{$user->profile->job_title}} at {{$user->company->name}} <a href="#" class="edit-ancor"><i
+                                    class="fas fa-pencil-alt edit-fontawsom"></i></a></h6>
+                        <p class="text-secondary user-profile-city  hover-edit">{{$user->profile->city->name}}
+                            , {{$user->profile->country->name}} <a href="#" class="edit-ancor"><i
+                                    class="fas fa-pencil-alt edit-fontawsom"></i></a></p>
+
+                        @foreach($user->skills()->get() as $index=>$us )
+                            @if($index<5)
+                            <span class="badge badge-info mb-1 px-2 py-1 hover-edit">{{$us->name}}</span>
+                            @endif
+                        @endforeach
+                        @foreach($user->links()->get() as $us)
+                            <div class="float-right">
+                                <a href="{{$us->url}}" class="m-3"><i class="fab fa-linkedin-in"></i></a>
+                                <a href="{{$us->url}}"><i class="fab fa-facebook-f"></i></a>
+                            </div>
+                        @endforeach
```

Comment:
This `foreach` will print this section
```html
<div class="float-right">
    <a href="{{$us->url}}" class="m-3"><i class="fab fa-linkedin-in"></i></a>
    <a href="{{$us->url}}"><i class="fab fa-facebook-f"></i></a>
</div>
```
8 Times corresponding to the number of URL for the user they are 
`linkedin` `facebook` `twitter` `nstagram` `behance` `github` `stackoverflow` `youtube` `blog` `website`